olp-cpp-sdk-dataservice-read/src/generated/parser/MessagesParser.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "MessagesParser.h"
+
+// clang-format off
```

Comment:
```suggestion
```
Why disabling format for the following 2 lines?

Reply:
Because if we enable clang-format on these 2 includes, the order of this includes will be swapped:
```
#include <olp/core/generated/parser/ParserWrapper.h>
#include "generated/parser/StreamOffsetParser.h"
```

Then, compiling this source file will result in a compile error, as the compiler won't be able to instantiate the `parse` template method with `model::StreamOffset` template argument

Reply:
Ouch, we'd better fix it otherwise we'll get plenty of head-banging troubleshooting sessions in future.
Are there solutions to this?

Reply:
Are you sure this is a problem? I saw other open source projects using the same `clang-format off` trick to overcome this C++ headers ordering issue.

Reply:
I'm just worried about maintenance. A minuscule change in the order headers are included can cause days of painful debugging. 
Anyway, it's off-topic to this change and should be discussed separately. I'll put a notice somewhere in the tickets/documentation.